### ISSUE TITLE
unused method parameter false positive on block arguments

### ISSUE THREAD

**Issue by ahfarmer** [oclint/oclint#244]
A screenshot is worth 1000 words:

![screen shot 2015-08-21 at 3 01 16 pm](https://cloud.githubusercontent.com/assets/597825/9420037/16c6e9c6-4816-11e5-9a90-f50380a14222.png)

**Comment by ryuichisaito6** [comment]
Duplicated for #135.

**Comment by ryuichis** [comment]
This is not a false positive after all. The correct way to define this should be

```objective-c
- (void)blockMethod: (void(^)(NSString *))block
```

Because `aString` is unused and impossible to be used.

Read #135 for discussions.
